### ISSUE TITLE
fix dropdown lists on alumni

### ISSUE THREAD

**Issue by irvin** [irvin/reps-portal-static-archive#2]
https://irvin.github.io/reps-portal-static-archive/people/alumni/

fix dropdown list of alumni page  at https://github.com/irvin/reps-portal-static-archive/tree/master/people/alumni
and pointed to pages under https://github.com/irvin/reps-portal-static-archive/tree/master/people/alumni

**Comment by Ryuno-Ki** [comment]
That view is paginated consisting of URLs of structure https://irvin.github.io/reps-portal-static-archive/people/alumni/?sort_key=&query=&page=<number> (up to 26 at this point in time)

**Comment by irvin** [comment]
@Ryuno-Ki I will try to fetch that, thanks.

**Comment by irvin** [comment]
uh, I forgot that we already had pages there. this issue is to fix the dropdown list of /alumni. (description clarified.)

**Comment by Ryuno-Ki** [comment]
Could you specify what dropdown? Is there a line of code?
(Using <kbd>y</kbd> will allow you to generate a permalink on GitHub)

Shall I open a new issue regarding removal of GA-Script?

**Comment by irvin** [comment]
> Shall I open a new issue regarding removal of GA-Script?

yes, please open. (but we will need to wait after syncing with the online site for last time)

**Comment by irvin** [comment]
![截圖 2020-04-22 下午6 43 37](https://user-images.githubusercontent.com/165936/79972942-4c342480-84c9-11ea-8bbe-e3a01b90c016.png)
this dropdown

**Comment by Ryuno-Ki** [comment]
Filled #4 regarding Google Analytics.

The dropdown is found in https://github.com/irvin/reps-portal-static-archive/blob/8993858195d6ff3349937105a951b224b5cf5d47/people/alumni/index.html#L615-L747 and respectively the other files (for reference)

**Comment by Ryuno-Ki** [comment]
Basically, a transformation of value from
`?sort_key=&amp;query=&amp;page=2` to `index_sort_key=&amp;query=&amp;page=2.html` is needed?

That is, replace `?` with `index_` and append a `.html`
